ccsdspy/utils.py

```diff
@@ -0,0 +1,66 @@
+"""Utils for the CCSDSPy package."""
+
+__author__ = "Daniel da Silva <mail@danieldasilva.org>"
+
+from io import BytesIO
+import warnings
+
+import numpy as np
+
+
+def split_by_apid(mixed_file, valid_apids=None):
+    """Split a stream of mixed APIDs into seperate streams by APID. 
+    
+    Parameters
+    ----------
+    mixed_file: str
```

Comment:
The type annotation of str isn't complete. On the next line you note in longer form English that it could be a file-like object. It should probably be noted on this line. Otherwise, people using PyCharm like me get a warning that we're passing in the wrong type.

Reply:
What is the convention for multiple types for PyCharm here? would `mixed_file: str, file-like` work?

Reply:
I think you can just type `mixed_file: str or file-like`. Now that I think about it, maybe PyCharm was inferring the type from the code instead of the docstring. I'm not sure. I tend to use [type hints](https://peps.python.org/pep-0484/) everywhere so it doesn't have to do much inference.